src/Utils/queryUtils.js

```diff
@@ -1,6 +1,8 @@
 const applyFilters = (query, filters) =>
   filters.reduce((accumulator, filter) => accumulator.where(filter.field, filter.condition, filter.value), query);
 
+const pipeOrderBy = (query, orderDirection, orderBy) => orderBy.reduce((accumulator, field) => accumulator.orderBy(field, orderDirection), query)
```

Comment:
I think you won't be able to order by more than one field at the same time. If you order by age you can't order by name :thinking:

Reply:
we should check if this is still not possible

Reply:
https://firebase.google.com/docs/firestore/query-data/order-limit-data @SKOLZ @LucasZibell